server/models/postgis/user.py

```diff
@@ -0,0 +1,66 @@
+#import geojson
+from server import db
+#from server.models.dtos.user_dto import UserDTO, UserFilterDTO, Pagination, UserSearchQuery, UserSearchDTO, \
+#     ListedUser
+from server.models.dtos.user_dto import UserDTO
+from server.models.postgis.lookups import UserRole
+from server.models.postgis.utils import NotFound, timestamp
+
+
+class User(db.Model):
+    """ Describes the user account table """
+    __tablename__ = "dmis_users"
+
+    user_id = db.Column(db.BigInteger, primary_key=True, index=True)
+    username = db.Column(db.String, unique=True, index=True)
+    role = db.Column(db.Integer, default=0, nullable=False)
+    email_address = db.Column(db.String)
+    password = db.Column(db.String)
+
```

Comment:
Might be useful to have a date_created column here that defaults to the timestamp